aidial_adapter_openai/utils/tokens.py

```diff
@@ -2,20 +2,24 @@
 Implemented based on the official recipe: https://cookbook.openai.com/examples/how_to_count_tokens_with_tiktoken
 """
 
-from typing import Any, List, Set, TypeVar
+from typing import Any, List
 
-from aidial_sdk.exceptions import (
-    InternalServerError,
-    TruncatePromptSystemAndLastUserError,
-    TruncatePromptSystemError,
-)
+from aidial_sdk.exceptions import InternalServerError
 from tiktoken import Encoding, encoding_for_model
 
 
 class Tokenizer:
+    """
+    Tokenizer for message.
+    Calculates only textual tokens, not image tokens.
+    """
+
     model: str
     encoding: Encoding
 
+    # Tokens that are added to all message tokens, not matter what
+    PROMPT_TOKENS = 3
```

Comment:
It's not prompt tokens per se, it's rather `_TOKENS_PER_REQUEST` - extra for a request. Similar to what we have for `tokens_per_message` and `tokens_per_name` - extra tokens for these entities.

Reply:
Renamed to `TOKENS_PER_REQUEST`, without `_T...` prefix, since I need this property outside of class